app/src/components/settings/user.tsx

```diff
@@ -28,6 +30,9 @@ export default function UserOptionsTab(props: any) {
                         <FormattedMessage defaultMessage="Find your API key here." description="Label for the link that takes the user to the page on the OpenAI website where they can find their API key." />
                     </a>
                 </p>
+                <p>
+                    <input type="checkbox" id="use-openai-whisper-api" checked={useOpenAIWhisper!} onChange={onUseOpenAIWhisperChange} /> Use the OpenAI Whisper API for speech recognition.
```

Comment:
This should use the Mantine `<Checkbox />` component for UI consistency:
https://mantine.dev/core/checkbox/

and the text next to it should be passed as the component's `label` prop